config/e3sm/config_files.xml

```diff
@@ -279,6 +280,7 @@
       <value component="mpaso"    >$SRCROOT/cime_config/allactive/config_pesall.xml</value>
       <value component="mali"     >$SRCROOT/cime_config/allactive/config_pesall.xml</value>
       <value component="mpassi"   >$SRCROOT/cime_config/allactive/config_pesall.xml</value>
+      <value component="ww3"      >$SRCROOT/cime_config/allactive/config_pesall.xml</value>
```

Comment:
Maybe this should be the time we start to move component specific layouts to their own file? since its a new component  If you have ones for ww3 standalone, they would go in $SRCROOT/components/ww3/cime_config/config_pes.xml

Reply:
Agreed, that probably makes sense.